assets/js/customize-preview.js

```diff
@@ -15,17 +15,21 @@
 			if ( isDark ) {
 				document.body.classList.add( 'is-dark-theme' );
 				document.documentElement.classList.add( 'is-dark-theme' );
+				document.body.classList.remove( 'is-light-theme' );
```

Comment:
We no longer need the is-light-theme class here, unless we want to add it for other people to use?

Reply:
Yeah, I'd rather keep it even if we don't use it... It will make customizations easier for those who will want to write custom css (`body.is-light-theme` is easier than `body:not(.is-dark-theme)`)